README.md

```diff
@@ -22,30 +22,24 @@ npm install activeui-migration
 
 `activeui-migration` requires two JSON files:
 
-- _content of ui.json_: the original content to migrate
+- _content.json_: the original content to migrate
 - _servers.json_: the ActivePivot server's information
 
-and a third optional file:
-
-- _content of pivot.json_: the content from the pivot folder on the content server.
-
 Let's create these files.
 
 ### Download the original content
 
-The content to migrate can be downloaded as a JSON file from the atoti+ Admin UI. If you are used to accessing ActiveUI at `https://my-server.com/ui/index.html`, then you should be able to access the Admin UI at `https://my-server.com/admin/ui/index.html`.
+The content to migrate can be downloaded as a JSON file from the atoti+ [Admin UI](https://activeviam.com/activepivot/6.0.0/docs/content_server/cs_overview/#admin-ui). If you are used to accessing ActiveUI at `https://my-server.com/ui/index.html`, then you should be able to access the Admin UI at `https://my-server.com/admin/ui/index.html`.
 
 :warning: Make sure to connect as a user with access to all the content that must be migrated. We recommend connecting as a user with the role `ROLE_CS_ROOT`.
 
-Right-click the folder named **ui** and click **Export**. This will download a file called _content of ui.json_. Move this file into your `migration` folder.
+Right-click at the root of the file tree, and click **Export**. This will download a file called _content.json_. Move this file into your `migration` folder.
 
 ![](/documentation/download-content.gif)
 
-### Optional: Download the content of the pivot folder
-
-:warning: If there are calculated measures to be migrated from ActiveUI-4 to the ActiveUI-5 content servers, this file is required. If this is not the case, this file can be disregarded.
-
-Right-click the folder named **pivot** and click **Export**. This will download a file called _content of pivot.json_. Move this file into your `migration` folder.
+:warning: Importing/exporting at the root of the file tree is available in atoti+ Admin UI version 5.1.0 and higher.
+If you don't see these actions in the context menu, it means that you are using an older Admin UI version.
```

Comment:
```suggestion
If you don't see these actions in the context menu, it means that you are using an older Admin UI version and will have to upgrade.
```
Just to make it clearer what you are referring to below when you say "you can do so...."